students/Ruohan/lesson03/slicing_lab.py

```diff
@@ -0,0 +1,53 @@
+'''fuctions of slicing exercise'''
+
+a_string = 'this is Ruohan'
+a_tuple = (12, 34, 44, 27, 56, 4, 19)
+
+
+def exchange_first_last(seq):
+    ''' return sequence with the first and last items exchanged'''
+    a_new_sequence = seq[-1:] + seq[1:-1] + seq[:1]
+    return a_new_sequence
+
+assert exchange_first_last(a_string) == 'nhis is Ruohat'
+assert exchange_first_last(a_tuple) == (19, 34, 44, 27, 56, 4, 12)
+
+
+
+def every_other_removed(seq):
+    '''return sequence with other item removed'''
+
+    if len(seq) > 1:
+        a_new_sequence = seq[::2]
+    return a_new_sequence
+assert every_other_removed(a_string) == 'ti sRoa'
+assert every_other_removed(a_tuple) == (12, 44, 56, 19)
+
+
+def first_last4_removed(seq):
+    '''return sequence with the first and last 4 items removed,
+    and every other item in between'''
+    if len(seq) > 4:
```

Comment:
Do you need this? What if you don't have this if condition but just return `seq[1:-4]` all the time?